docs/api/features/pcoords.rst

```diff
@@ -5,52 +5,54 @@ Parallel Coordinates
 
 Parallel coordinates is multi-dimensional feature visualization technique  where the vertical axis is duplicated horizontally for each feature. Instances are displayed as a single line segment drawn from each vertical axes to the location representing their value for that feature. This allows many dimensions to be visualized at once; in fact given infinite horizontal space (e.g. a scrolling window), technically an infinite number of dimensions can be displayed!
 
-Data scientists use this method to detect clusters of instances that have similar classes, and to note features that have high variance or different distributions. We can see this in action after first loading our occupancy classification dataset:
+Data scientists use this method to detect clusters of instances that have similar classes, and to note features that have high variance or different distributions. We can see this in action after first loading our occupancy classification dataset. Note that the visualization can be drawn with either the ``ParallelCoordinates`` visualizer or using the ``parallel_coordinates`` quick method:
```

Comment:
Tried my best to make this as fluid as possible.  the "Note that the visualization can be drawn..." was previously used between split inline code.

Reply:
Yeh, that's tricky - but thanks for working it in! Another option with the documentation is to use a callout. E.g.

```
.. note:: These visualizations can be produced with either the ``ParallelCoordinates`` visualizer or by using the `parallel_coordinates`` quick method.
```

Which renders similarly to:

![screenshot 2019-02-06 14 02 58](https://user-images.githubusercontent.com/745966/52366510-ed939f80-2a17-11e9-94ef-32237696087e.png)

I don't know if that's better here, but it is an option.

Reply:
I will try to incorporate the note directive..let me know what you think

Reply:
great suggestions...now I have to decide if I should place before the first plot directive or after